### DISCUSSION TITLE
Customize location of toolbar

### DISCUSSION THREAD

**Discussion by rbsl-yasmin** [codex-team/editor.js#1927] [Q&A]
I wanted to toolbox/toolbar to be opened all the time and it should be displayed at the top of the editor. like that
![image](https://user-images.githubusercontent.com/20413818/126035589-efef6e40-f18d-402f-9cfe-9d2b625e6697.png)
instead of clicking on +, the toolbar should always be opened at the top of the editor.

is there a way to modify the location of the toolbar?
Thanks

**Comment by luyang12138** [comment]
+1

**Comment by finaly55** [comment]
+1
